_includes/ga.html

```diff
@@ -0,0 +1,11 @@
+<script>
```

Comment:
Especially because this site is likely going to be used by lots of folks new to Jekyll, I'd spell out `google-analytics` in the filename, so it's less opaque what's going on.